collection-matchers/src/test/java/ru/yandex/qatools/matchers/collection/WrapperTest.java

```diff
@@ -0,0 +1,49 @@
+package ru.yandex.qatools.matchers.collection;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Objects;
+
+/**
+ * @author Vladislav Bauer
+ */
+
+public class WrapperTest {
+
+    @Test
+    public void testEquals() {
+        Wrapper<Object> objectWrapper1 = createTestWrapper();
+        Wrapper<Object> objectWrapper2 = createTestWrapper().wrap(objectWrapper1);
+        Wrapper<Object> objectWrapper3 = createTestWrapper().wrap(objectWrapper1);
+
+        //noinspection EqualsBetweenInconvertibleTypes
+        Assert.assertFalse(objectWrapper1.equals(new Object()));
```

Comment:
please add message to assertions, as of it like a smell 'assertion roulette' - we don't know which fail until look at code.

also please use `assertThat + equalTo` or `assertThat + not` form for assertions as more verbosive and refactoring friendly